sbws/core/scanner.py

```diff
@@ -261,20 +263,25 @@ def measure_relay(args, conf, destinations, cb, rl, relay):
                 ]
     # Pick a destionation
     dest = destinations.next()
-    # If there is no any destination at this point, it can not continue.
+    # When there're no any functional destinations.
     if not dest:
-        # XXX: this should return a ResultError
-        # instead of stopping the scanner once a destination can be recovered.
+        # NOTE: When there're still functional destinations but only one of
```

Comment:
I think there is missing some binding word? "When there's still an available functional destination, but ..." ?